tests/cypress/e2e/model-spec.cy.js

```diff
@@ -0,0 +1,128 @@
+
+        // Hide System Security 1 and make System Security 2 visible
+        assertHiddenAndVisibleToggled('ss-q1', 'ss-q2');
+        cy.get('.box-body #questions-section #questions-buttons .button-row .btn-next').click({force: true})
+
+        // Hide System Security 2 and make System Security 3 visible
+        assertHiddenAndVisibleToggled('ss-q2', 'ss-q3');
+        cy.get('.box-body #questions-section #questions-buttons .button-row .btn-next').click({force: true})
+
+        // Hide System Security 3 and make System Security 4 visible
+        assertHiddenAndVisibleToggled('ss-q3', 'ss-q4');
+        cy.get('.box-body #questions-section #questions-buttons .button-row .btn-next').click({force: true})
+
+        // Hide System Security 4 and make Information Management 1 visible
+        assertHiddenAndVisibleToggled('ss-q4', 'im-q1');
+        cy.get('.box-body #questions-section #questions-buttons .button-row .btn-next').click({force: true})
+
+        // Hide Information Management 1 and make Information Management 2 visible
+        assertHiddenAndVisibleToggled('im-q1', 'im-q2');
+        cy.get('.box-body #questions-section #questions-buttons .button-row .btn-next').click({force: true})
+
+        // Hide Information Management 2 and make Information Management 3 visible
+        cy.get(`.box-body #questions-section #questions .question .question-content .question-text #im-q2`)
+            .parent().parent().parent().should('have.class', 'question hidden')
+        cy.get(`.box-body #questions-section #questions .question .question-content .question-text #im-q3a`).should('be.visible')
+        cy.get(`.box-body #questions-section #questions .question .question-content .question-text #im-q3a`)
+            .parent().parent().parent().parent().should('have.class', 'question')
+        cy.get('.box-body #questions-section #questions-buttons .button-row .btn-next').click({force: true})
+
+        // Hide Information Management 3 and make Technical Skills visible
+        cy.get(`.box-body #questions-section #questions .question .question-content .question-text #im-q3a`)
+            .parent().parent().parent().parent().should('have.class', 'question hidden')
+        cy.get(`.box-body #questions-section #questions .question .question-content .question-text #ts-q1a`).should('be.visible')
+        cy.get(`.box-body #questions-section #questions .question .question-content .question-text #ts-q1a`)
+            .parent().parent().parent().parent().should('have.class', 'question')
+
+        //Finish button visible, click i
+        cy.get('.box-body #questions-section #questions-buttons .button-row .btn-finish').should('be.visible')
+        cy.get('.box-body #questions-section #questions-buttons .button-row .btn-finish').click({force: true})
+        cy.get('#models-section-container').should('be.visible')
+
+    })
+
+    function assertHiddenAndVisibleToggled(toHide, toShow) {
```

Comment:
`toHide` and `toShow` imply that these elements will be hidden and shown in the function; should they just be `hidden` and `visible`?